app/src/androidTest/java/com/github/se/polyfit/ui/screen/LoginTest.kt

```diff
@@ -0,0 +1,46 @@
+package com.github.se.polyfit.ui.screen
+
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.performClick
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.se.polyfit.MainActivity
+import com.kaspersky.kaspresso.testcases.api.testcase.TestCase
+import io.github.kakaocup.compose.node.element.ComposeScreen
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class LoginTest : TestCase() {
+  @get:Rule val composeTestRule = createAndroidComposeRule<MainActivity>()
+
+  @Test
+  fun titleAndButtonAreCorrectlyDisplayed() {
+    ComposeScreen.onComposeScreen<LoginScreen>(composeTestRule) {
+      // Test the UI elements
+      loginTitle {
+        assertIsDisplayed()
+        assertTextEquals("Welcome")
+      }
+      loginButton {
+        assertIsDisplayed()
+        assertHasClickAction()
+      }
+    }
+  }
+
+  @Test
+  fun loginButtonNavigatesToNewScreen() {
+    // Launch the LoginScreen
+    ComposeScreen.onComposeScreen<LoginScreen>(composeTestRule) {
+      // Test the UI elements
+      loginButton {
+        assertIsDisplayed()
+        assertHasClickAction()
+        performClick()
+      }
+      // TODO check we are on another page
```

Comment:
If it's not too difficult, it might be a good idea to test both the successful and unsuccessful cases of a login attempt, (possibly through a mock)